mapswipe_workers/mapswipe_workers/definitions.py

```diff
@@ -12,6 +12,9 @@
     os.makedirs(DATA_PATH)
 LOGGING_FILE_PATH = os.path.join(DATA_PATH, "mapswipe_workers.log")
 
+# number of geometries for project geometries
+MAX_INPUT_GEOMETRIES = 2
```

Comment:
should we change this to 10? how many geometries do we want to allow to start with?